packages/components/src/list/index.js

```diff
@@ -83,7 +84,7 @@ class List extends Component {
 								) }
 								<div className="woocommerce-list__item-text">
 									<span className="woocommerce-list__item-title">
```

Comment:
Since `<Text>` will render a `p` tag, we should probably update this to a `div` to be more semantic.

Side note that it would be nice if `Text` allowed `span` in the `as` prop and/or a `className` prop as well, but those are obviously upstream issues.